linkerd/failure-accrual/src/test/scala/io/buoyant/linkerd/failureAccrual/ConfigTest.scala

```diff
@@ -0,0 +1,119 @@
+package io.buoyant.linkerd.failureAccrual
+
+import com.twitter.finagle.service.Backoff
+import com.twitter.util.Duration
+import io.buoyant.config.Parser
+import io.buoyant.linkerd.{ConstantBackoffConfig, FailureAccrualConfig, JitteredBackoffConfig}
+import io.buoyant.test.FunSuite
+import org.scalatest.{Matchers, OptionValues, OutcomeOf}
+import com.twitter.conversions.time._
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.prop.{PropertyChecks, TableDrivenPropertyChecks}
+import org.scalacheck.Gen
+import org.junit.runner.RunWith
+
+/**
+ * Created by eliza on 6/27/17.
+ */
+@RunWith(classOf[JUnitRunner])
```

Comment:
This [should](https://github.com/linkerd/linkerd/pull/1451/files/af864d60f3b6dcb11f39776ba5ccda51a15683eb#diff-870b1825491434809bccc89f7d11fe56R7) already extend `io.buoyant.test.FunSuite`. I think you're right that the annotation can be removed, though.